### PR TITLE
Use grpc in the status component

### PR DESCRIPTION
**Pull request by liyangguang**
- Use grpc in the status component
- Add project.config.js back to gitignore. And added the config structure to the README.
- A few lines of controller code on the local machine.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +0,0 @@
-.*
```
Comment (bowenz-google) [inline .gitignore]: Why delete this file?

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +0,0 @@
-[submodule "third_party/grpc-web"]
```
Comment (bowenz-google) [inline .gitmodules]: We need grpc-web submodule for web streaming.
Not sure when gflags was added.

File: .gitignore (hunk 1 above)
Comment (liyangguang) [reply .gitignore]: Not intended to. Don't know why there's a ".*" in .gitignore, which messed up these two files. Removing this line.

File: .gitmodules (hunk 2 above)
Comment (liyangguang) [reply .gitmodules]: Not intended to make changes. Added back.

**Pull request merged**